app/src/main/java/org/coepi/android/ui/alerts/AlertsViewModel.kt

```diff
@@ -63,17 +68,38 @@ class AlertsViewModel(
      * This function parses [Alert] objects received from the [AlertsRepo] into readable strings that get displayed
      * in the recycler view item_alert views bound by the [AlertsAdapter]
      */
-    private fun Alert.toViewData(): AlertViewData =
+    private fun Alert.toViewData(alerts: List<Alert>): AlertViewData =
         AlertViewData(
             exposureType = listOfNotNull(
                 report.coughSeverity.toUIString(),
                 toBreathlessnessString(report.breathlessness),
                 report.feverSeverity.toUIString()
             ).joinToString("\n"),
             contactTime = DateFormatters.hourMinuteFormatter.formatTime(contactTime.toDate()),
+            contactTimeMonth = DateFormatters.monthDayFormatter.formatMonthDay(contactTime.toDate()),
+            showMonthHeader = showMonthHeader(alerts),
             report = this
         )
 
+    private fun Alert.showMonthHeader(alerts: List<Alert>): Boolean {
```

Comment:
Hmm, as mentioned, I don't like the idea of having the title in the alert items. But if this is working correctly we can pass it for now and I'll refactor it later. Here's the [iOS implementation](https://github.com/Co-Epi/app-ios/blob/develop/CoEpi/ui/alerts/AlertsViewModel.swift#L47) btw, though it can't be used as reference 1:1 since we have to organize the sections there hierarchically.